webapp/src/ts/modules/analytics/analytics-target-aggregates-sidebar-filter.component.html

```diff
@@ -0,0 +1,40 @@
+<section class="sidebar-filter">
+  <div class="sidebar-backdrop" [ngClass]="{ hidden: !isOpen }" (click)="toggleSidebarFilter()"></div>
+
+  <div class="sidebar-main" [ngClass]="{ hidden: !isOpen }">
+    <div class="sidebar-header">
+      <p class="sidebar-title">Filter</p>
+      <a class="sidebar-close fa fa-times" (click)="toggleSidebarFilter()"></a>
+    </div>
+
+    <div class="sidebar-body">
+      <mat-accordion multi>
+        <div class="facility-filter-wrap">
+          <div class="facility-filter-title">
+            <span class="facility-title"> {{ facilityLabel | translate }} </span>
+          </div>
+          <div class="facility-filter-body">
+            <div *ngFor="let facility of userFacilities" class="facility-filter-place">
```

Comment:
The double wrap might be unnecessary here. You can use it like this: 
`<ng-container *ngFor=".....`